.travis.yml

```diff
@@ -17,7 +17,7 @@ notifications:
     on_success: change
     on_failure: always
 script:
-- bundle exec rake default
+- bundle exec rake quick
```

Comment:
the difference between default and quick is that it will run all kitchen tests. As we want to only run the tests with the current ruby version we use quick and the line below to run it for only the appropriate version.